WordPress/Classes/ViewRelated/System/Coordinators/MySitesCoordinator.swift

```diff
@@ -27,10 +32,15 @@ class MySitesCoordinator: NSObject {
         prepareToNavigate()
     }
```

Comment:
seems that this method is only calling another one, perhaps we can just use the inner one?

Reply:
Addressed in my latest commit.